integration_test.go

```diff
@@ -109,6 +112,36 @@ func TestOpenAPIGeneration(t *testing.T) {
 			},
 			wantFiles: []string{"test7/openapiv3.yaml"},
 		},
+		{
+			name:       "Test no markers are ignored when ignored_kube_markers is zero length",
+			id:         "test8",
+			perPackage: false,
+			genOpts:    "yaml=true,single_file=true,proto_oneof=true,int_native=true,multiline_description=true,disable_kube_markers=false,ignored_kube_marker_substrings=",
+			inputFiles: map[string][]string{
+				"test8": {"./testdata/test8/markers.proto"},
+			},
+			wantFiles: []string{"test8/openapiv3.yaml"},
+		},
+		{
+			name:       "Test ignored_kube_markers option ignores a single marker",
+			id:         "test9",
+			perPackage: false,
+			genOpts:    "yaml=true,single_file=true,proto_oneof=true,int_native=true,multiline_description=true,disable_kube_markers=false,ignored_kube_marker_substrings=Required",
+			inputFiles: map[string][]string{
+				"test9": {"./testdata/test9/markers.proto"},
+			},
+			wantFiles: []string{"test9/openapiv3.yaml"},
+		},
+		{
+			name:       "Test ignored_kube_markers option ignores multiple markers",
+			id:         "test10",
+			perPackage: false,
+			genOpts:    "yaml=true,single_file=true,proto_oneof=true,int_native=true,multiline_description=true,disable_kube_markers=false,ignored_kube_marker_substrings=Required+example",
```

Comment:
Is `example` an existing marker? If not, we should test against a supported marker.

Reply:
This is an existing marker, it exists in test8,9,10 and was duplicated from test6